lib/theme/app_theme.dart

```diff
@@ -9,6 +9,24 @@ class AppTheme {
           primary: AppColors.blackRussian,
         ),
         fontFamily: FontFamily.neuzeit,
+        elevatedButtonTheme: ElevatedButtonThemeData(
+          style: ElevatedButton.styleFrom(
+            backgroundColor: Colors.white,
+            foregroundColor: AppColors.blackRussian,
+            textStyle: const TextStyle(
+              /** Regular / Semi Bold */
```

Comment:
This comment is confused and does not follow comment formatting convention (use `//` for single line)

Reply:
The comment is to indicate the name of the style used in Figma 🙏 

I'll change it to `//`